### PR TITLE
Colored power button

### PR DESCRIPTION
**Pull request by kohrar**
The power button is colored:
* Red with a / over when off
* Green when on

### CODE REVIEW

File: opendps/opendps.c [hunk 1]
```diff
@@ -718,14 +724,17 @@ void opendps_update_wifi_status(wifi_status_t status)
...
-    }
+	is_enabled = enabled;
+
+	if (is_enabled) {
+		tft_blit((uint16_t*) gfx_poweron,
+				GFX_POWERON_WIDTH, GFX_POWERON_HEIGHT,
+				TFT_WIDTH-GFX_POWERON_WIDTH, TFT_HEIGHT-GFX_POWERON_HEIGHT);
+	} else {
+		tft_blit((uint16_t*) gfx_poweroff,
+				GFX_POWERON_WIDTH, GFX_POWERON_HEIGHT,
```
Comment (JackNewman12) [inline opendps/opendps.c]: I think this should be poweroff width/height. Shouldn't matter

Comment (kohrar) [reply opendps/opendps.c]: Yup. Good catch. Copy and paste strikes again.

**Comment by JackNewman12** [comment]
I really like the green power icon (I actually made the same change on my fork). 

I personally prefer to have it just blank rather than red for power off. (Any color = something is live)
Would it make sense to add this as a compile time switch?

**Comment by kohrar** [comment]
I added two flags to the Makefile:
* `POWER_OFF_VISIBLE`
* `POWER_COLORED`

Setting `POWER_COLORED=1` and `POWER_OFF_VISIBLE=0` (defaults now) should get your desired behavior.

**Pull request merged**